include/OgreOggSoundPrereqs.h

```diff
@@ -49,7 +48,7 @@
  * 1 - Enable EFX suport
  */
 #	ifndef HAVE_EFX
-#		define HAVE_EFX 1
+#		define HAVE_EFX 0

```

Comment:
is this change on purpose, or was it accidentally included?

Reply:
Sorry,
It does not a change on purpose, it is my fault on comitting.
Thanks, I did not noticed that.

Reply:
I changed one dependency from OpenAL to OpenALSoft, and I do not know, why OpenALSoft does not suport EFX.
But CMakeList.txt must work.